host/host.go

```diff
@@ -99,7 +99,10 @@ func isSupportedSyzkall(c *sys.Call) bool {
 		_, err := os.Stat("/dev/fuse")
 		return err == nil && syscall.Getuid() == 0
 	case "syz_emit_ethernet":
-		_, err := os.Stat("/dev/net/tun")
+		fd, err := syscall.Open("/dev/net/tun", syscall.O_RDWR, 0)
+		if fd != -1 {
```

Comment:
сheck `err == nil`
if err != nil, other return values are generally undefined (usually zero)
it may work now (I didn't check), but can break in future